NEWS.txt

```diff
@@ -116,6 +116,8 @@ New features
       table.
     - Added ability to invalidate auth caches through corresponding `nodetool` commands and virtual tables.
     - DCL statements in audit logs will now obscure only the password if they don't fail to parse.
+    - Added API for alternative memtable implementations. For details, see
+      src/java/org/apache/cassandra/db/memtable/Memtable_API.md
```

Comment:
Nit: I think this entry should be at the beginning of the list of new features, like in a change log. Also, we might want to mention than no alternative implementation has been added, to avoid confusions.

Reply:
Done